### PR TITLE
[SC-650] Bug(84): Redo - Update contract dependencies to upgradeable versions

### PR DESCRIPTION
**Pull request by marvinkruse**
Redoing PR #571 because of some issues with the ElasticReceiptToken. Anything regarding the ElasticReceiptToken will be handled in a separate PR.

### CODE REVIEW

File: src/external/forwarder/TransactionForwarder_v1.sol [hunk 1]
```diff
@@ -34,9 +34,6 @@ contract TransactionForwarder_v1 is
     ERC2771Forwarder,
     Context
 {
```
Comment (Zitzak) [inline src/external/forwarder/TransactionForwarder_v1.sol:36]: We are we removing the `__gap` here. Don't we need it to for the possible upgrade?

**Review by Zitzak — COMMENTED** [review]
LGTM, only unsure why the gap was removed

**Comment by marvinkruse** [comment]
> LGTM, only unsure why the gap was removed

I removed it there as it's not an upgradeable contract, and thus doesn't serve any purpose there.

File: src/external/forwarder/TransactionForwarder_v1.sol (hunk 1 above)
Comment (marvinkruse) [reply src/external/forwarder/TransactionForwarder_v1.sol:36]: It's a non upgradable contract, at least thats how we built it

Comment (Zitzak) [reply src/external/forwarder/TransactionForwarder_v1.sol:36]: Thank you, I'll approve

**Review by Zitzak — APPROVED** [review]

**Pull request merged**